### DISCUSSION TITLE
No face recognition at all

### DISCUSSION THREAD

**Discussion by slashgob** [immich-app/immich#8056] [Q&A] [answered]
### The bug

I can see the photos, the map and location is working, places shows location but I get zero face recognition. I have tried rescanning for faces and run the detection ahain, 0 waiting but still no face detection showing.

### The OS that Immich Server is running on

Unraid

### Version of Immich Server

1.98.2

### Version of Immich Mobile App

1.98.2.128

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
not sure how to get this?
```

### Your .env content

```Shell
where is this?
```

### Reproduction steps

```bash
it isn't triggered, it just doesn't work.
```

### Additional information
Intel® Core™ i7-3770K CPU @ 3.50GHz
NVIDIA GeForce GTX 1050

**Comment by martabal** [comment]
Are you using the AIO version ?

**Comment by slashgob** [comment]
I'm using the ghcr.io/imagegenius veriosn. I have Redis and tensorchord/pgvecto-rs also installed. I'd be happy to share some logs if I know how to get to them?

**Comment by martabal** [comment]
Are you using the noml version ?

**Comment by slashgob** [comment]
> Are you using the noml version ?

I don't know what that is?

**Comment by martabal** [comment]
The image tag

**Comment by slashgob** [comment]
no the repository is : ghcr.io/imagegenius/immich

but after reading, it is suggesting I need to install Nvidia Container Toolkit in the host OS?

**Comment by martabal** [comment]
Looks fine, can you share your docker logs and your config (environment variables and volumes) ?

**Comment by slashgob** [comment]
> Looks fine, can you share your docker logs and your config (environment variables and volumes) ?

I do not know how to get this information from Unraid?

**Comment by martabal** [comment]
I think you can export the template, but screenshots are good

**Comment by slashgob** [comment]
Had to split it into 2 screenshots...

1.
![image](https://github.com/immich-app/immich/assets/44810039/159517cc-58a9-4c18-be91-b41048ced326)

2.
![image](https://github.com/immich-app/immich/assets/44810039/93155fe8-c08a-4d70-902e-816e5f0188ea)

**Comment by martabal** [comment]
Make sure to follow the instructions if you want to use hardware acceleration for machine learning https://github.com/imagegenius/docker-immich?tab=readme-ov-file#nvidia-hardware-acceleration

Other than that, machine-learning should work, can you share your docker logs ?

**Comment by slashgob** [comment, marked as answer]
Adding the below fixed the issue for me:

Extra Parameters (Advanced View): --runtime=nvidia
MACHINE_LEARNING_GPU_ACCELERATION (Variable): cuda
MACHINE_LEARNING_WORKERS (Variable): 1 (I left at default)

Create these extra variables:

NVIDIA_VISIBLE_DEVICES (Variable): all
NVIDIA_DRIVER_CAPABILITIES (Variable): all
